### ISSUE TITLE
咨询您一个问题

### ISSUE THREAD

**Issue by lyhwyl** [tianlizhang/OKDPH#2]
最近阅读了您的这篇‘Generalization Matters: Loss Minima Flattening via Parameter Hybridization for Efficient Online Knowledge Distillation’论文，对于这篇论文，我深受启发，于是进行了复现。在复现过程中，因为imagenet数据集比较庞大，于是我使用了Tiny-imagenet数据集（我已经将其格式和imagenet数据集的格式进行了调整）进行了复现，所以我想请问您在使用这个数据集上进行复现实验时，是否设置了一些参数还是修改了什么地方，如果能告知，我将不胜感谢。

**Comment by tianlizhang** [comment]
For the settings of ImageNet, we employ the standard ResNet18 as the backbone and train 100 epochs with a learning rate of 0.1, which is adjusted by the multi-step scheduler with the gamma of 0.1 and milestones of {30, 60, 90} to obtain faster convergence. The weight decay of the SGD optimizer and batch sizes are set to 1e−4 and 256, respectively. The two student models and the HWM in our OKDPH are trained with three different transform composition: RandomCrop+RandomHorizontalFlip, RandomCrop+RandomAugment, and RandomCrop+AutoAugment, resepectively. Before data augmentation, all input images are resized to the same size of pixels, where 224×224 on ImageNet.

In addition, the attachment contains supplementary materials for this paper, hoping to be helpful to you!
[CVPR2023_OKDPH__Supplementary_Materials.pdf](https://github.com/tianlizhang/OKDPH/files/14632508/CVPR2023_OKDPH__Supplementary_Materials.pdf)
